bin/main.rb

```diff
@@ -1,4 +1,97 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-puts 'Hello, world!'
+class Game
+  require_relative '../lib/player'
+  require_relative '../lib/board'
+  def initialize
+    @player1 = nil
+    @player2 = nil
+    @board = nil
+
+    # Greetings start
+    puts 'Hello! Welcome, Lets PLAY TIC TAC TOE '
+
+    # First player name (repeat if empty):
+    while @player1.nil? || @player1.empty?
+      print 'What is your name Player One: '
+      @player1 = gets.chomp
```

Comment:
[Optional]
We could add `strip` to also avoid empty string with only white spaces.
Note that `'   '.empty? == false while '   '.strip.empty? == true`

```suggestion
      @player1 = gets.chomp.strip
```